src/Button.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = ({ clicked, value }) => <button onClick={clicked}>{value}</button>;
+
+Button.propTypes = {
+  clicked: PropTypes.func.isRequired,
+  value: PropTypes.string
```

Comment:
kérem legyen ez is required, elég hülyén néz ki a gomb, ha nem írsz be neki semmit, köfi!